loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -571,10 +572,33 @@ static void binary_update_new_binary_test(void)
 	unlink(filepath);
 }
 
-static void binary_update_run_tests(int repetition_num)
+static void binary_update_byitself_test(void)
 {
-	printf("\n** Binary Update Example %d-th Iteration.\n", repetition_num);
+	int ret;
+	char filepath[BINARY_PATH_LEN];
+	binary_update_info_t bin_info;
+
+	ret = binary_update_getinfo(MYAPP_NAME, &bin_info);
+	if (ret != OK) {
+		return;
+	}
+
+	/* Copy current binary and Write invalid crc. */
+	ret = binary_update_download_binary(&bin_info, DOWNLOAD_VALID_BIN);
```

Comment:
You passed `DOWNLOAD_VALID_BIN`, but in the comment `Write invalid crc`. Which one is correct?